### ISSUE TITLE
bug(datepicker): inconsistent announcement of start date and end

### ISSUE THREAD

**Issue by zarend** [angular/components#23445]
copied from internal bug report: http://b/195600297

1. Browse the Datepicker demo URL
2. Navigate to the "Date range selection" section
3. Activate "Open calendar" button from date range section, say, "Date range selection"
4. Choose any start date and end dates
5. Focus again on the start date and end date text boxes

Actual result:
1. It is not announcing the state date on state date edit box with date using VoiceOver, NVDA and JAWS with linear navigation
2. It is announcing "Enter a date range, edit text" and then with a long pause of 2-3 seconds & announcing "start date", where as it is announcing "End date, Edit text, Enter a date range" when focus on End Date editbox using ChromeVox

EXPECTED result:
It should announce consistently on both start date and end date edit boxes. Also, there should not be delay in announcing the start date

**Comment by zarend** [comment]
This was fixed by #25457
